### ISSUE TITLE
ESP32 Menuconfig LV_CONF_SKIP always YES

### ISSUE THREAD

**Issue by SinglWolf** [kisvegabor/lvgl#2562]
When starting Menuconfig, there is no selection line to disable LV_CONF_SKIP.
As a result, LV_CONF_SKIP is always YES.
![2021-09-17_08-05-09](https://user-images.githubusercontent.com/32172969/133718539-40877c61-bdab-494c-896f-5a3f601485ad.png)
Can fix it like this?
![2021-09-17_08-10-09](https://user-images.githubusercontent.com/32172969/133718622-45ef5f0f-423a-4929-bdeb-8785b6819f9d.png)

```
    config LV_CONF_MINIMAL
        bool "LVGL minimal configuration."

    config LV_CONF_SKIP
        bool "SKIP LV_CONF"
        default y
        help
            Define CONFIG_LV_CONF_SKIP so we can use LVGL
            without lv_conf.h file, the lv_conf_internal.h and
            lv_conf_kconfig.h files are used instead.


    menu "Color settings"
```

**Comment by kisvegabor** [comment]
Basically is it a misformatted Kconfig option? 

cc @C47D

**Comment by C47D** [comment]
When adding the `"SKIP LV_CONF"` string after the option type (`bool`) we expose the variable to be changed. Without it, the user doesn't see the option.

Having the description of it on the help section instead of a comment is much better tho. 

IIRC we were forcing the usage of Kconfig to use it with esp32.

**EDIT**
As far as I know, we haven't tried to configure LVGL with a `lv_conf.h` file instead of using Kconfig

**Comment by kisvegabor** [comment]
> When adding the "SKIP LV_CONF" string after the option type (bool) we expose the variable to be changed. Without it, the user doesn't see the option.

Then why isn't it visible for @SinglWolf ?

**Comment by C47D** [comment]
Because before changing it there wasn't any string after the symbol type.
I think that string has its name, but I can't remember it right now.

**Comment by kisvegabor** [comment]
Ah, got it now. Do you agree to fix it as @SinglWolf suggested?

**Comment by C47D** [comment]
@SinglWolf do you plan to use `lv_conf.h` instead of Kconfig for LVGL configuration?. 

@kisvegabor 
If so we can test the suggestion before the change, let's remember the Kconfig file is used in other projects out of LVGL control, such as Nuttx and Zephyr

**Comment by SinglWolf** [comment]
> @SinglWolf do you plan to use `lv_conf.h` instead of Kconfig for LVGL configuration?.

If not using Kconfig would shorten the recompilation time of the project when making changes to lv_conf.h, then definitely yes.

**Comment by C47D** [comment]
I guess it would, but I haven't tried it tho.

**Comment by kisvegabor** [comment]
> If so we can test the suggestion before the change, let's remember the Kconfig file is used in other projects out of LVGL control, such as Nuttx and Zephyr

If the users don't touch it shouldn't cause an trouble, right?

**Comment by kisvegabor** [comment]
Working on it in #2555.

**Comment by kisvegabor** [comment]
Will be fixed in #2555
